app/gateway/configuration.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Kong Gateway Configuration Reference
```

Comment:
Can we tell Vale to ignore titles? Looks like you can't use variables in titles, it breaks the build.